account_journal_activate_tracking/models/account_journal.py

```diff
@@ -0,0 +1,12 @@
+from odoo import models, fields
+
+class AccountJournal(models.Model):
+    _inherit="account.journal"
+    
+    name = fields.Char(string='Journal Name', required=True, tracking=True)
```

Comment:
When adding a new attribute to an existing field, no need to repeat existing attributes, simply do this:
```suggestion
    name = fields.Char(tracking=True)
```
Odoo will add new field attribute and preserve existing ones.